### ISSUE TITLE
Remove contributor names from translated strings?

### ISSUE THREAD

**Issue by dorkster** [flareteam/flare-engine#1726]
Names of people in the credits cutscenes are typically not something that needs to be translated. yet our translation script picks them up regardless.

A Transifex user suggested adding a `text_nt` property to cutscenes to flag a string as "no translate". Cutscenes are the primary, and possible sole, use case here. So it's likely that an engine-wide syntax to do this would be overkill.

**Comment by McSinyx** [comment]
Please do! Some names look quite like a valid English word that might confuse translator. Also could you please change my name (Nguyễn Gia Phong) to the ASCII alias McSinyx? I wasn't aware of the encoding problem when you asked.

**Comment by dorkster** [comment]
So instead of adding a new engine property, I've opted to simply [extend](https://github.com/dorkster/flare-engine/commit/e1a38dd401e509f960fde67f2a3329769ba9c1f7) the `xgettext.py` script to exclude everything but a fixed set of strings when reading a credits cutscene file. This is a much cleaner solution as it requires no changes to the game data.

I also changed your name in the credits to McSinyx as requested.
